draft-dnsop-deleg.md

```diff
@@ -149,60 +147,23 @@ NS based delegation supports DNS over UDP and TCP, but does not have the ability
-
-    example.com.  86400  IN DELEG 0   config2.example.net.
-    example.com.  86400  IN NS     ns2.example.net.
-
-The example.net authoritative server may return the following SVCB records in response to a query as directed by the above records.
-
-    config2.example.net 3600    IN SVCB . (
-                    ipv4hint=192.0.2.54,192.0.2.56
-                    ipv6hint=2001:db8:2423::3,2001:db8:2423::4 )
-
-The above records indicate to the client that the actual configuration for the example.com zone can be found at config2.example.net
-
-Later sections of this document will go into more detail on the resolution process using these records.
-
-## Goal of the DELEG record
-
-The primary goal of the DELEG records is to provide zone owners a method to signal capabilities to clients how to connect and validate a subdomain. This method coexists with NS records in the same zone. 
-
-The DELEG record is authoritative in the parent zone and, if signed, has to be signed with the key of the parent zone. The target of an alias record is an SVCB record that exists and can be signed in the zone it is pointed at, including the child zone.
-
-## DNSSEC is RECOMMENDED
-
-While DNSSEC is RECOMMENDED, unsigned DELEG records may be retrieved in a secure way from trusted, Privacy-enabling DNS servers using encrypted transports.
-
-FOR DISCUSSION: This will lead to cyclical dependencies. A DELEG record can introduce a secure way to communicate with trusted, Privacy-enabling DNS servers. For that, it needs to be DNSSEC signed. 
+# DELEG Record Type
 
-### Preventing downgrade attacks
+The DELEG record wire format is the same as the SVCB record defined in {{?RFC9460}}, but with a different value for the resource record type of TBD. The IANA registry for for further Service Parameter Keys that might be needed also is the one used for SVCB.
 
-A flag in the DNSKEY record is used as a backwards compatible, secure signal to indicate to a resolver that DELEG records are present or that there is an authenticated denial of a DELEG record. Legacy resolvers will ignore this flag and use the DNSKEY as is.
+The SVCB record allows for two types of records, the AliasMode and the ServiceMode. The DELEG record uses both. The Target Name either points to a set of name servers answering for the delegated domain if used in ServiceMode or to an SVCB record in AliasMode that then has to be interpreted further to get to the actual name server. The AliasMode DELEG record might point to another AliasMode SVCB record or a CNAME. In order to not allow unbound indirection of DELEG records the maximum number of indirections, CNAME or AliasMode SVCB is 4. The SvcPriority field either can be 0 for AliasMode or 1 for ServiceMode. Different priorities are not used for DELEG delegations. 
```

Comment:
Should 4 be recommended? It feels very arbitrary.

Reply:
I realize this was already present. I think the question still stand, but we should probably address it separately.

Reply:
Keeping it 4 for now.